ferrocene/doc/sphinx-shared-resources/src/ferrocene_standalone_make_cli.py

```diff
@@ -32,6 +33,9 @@ def build_docs(root, builder, clear, serve, debug):
         # Enable parallel builds:
         args += ["-j", "auto"]
     if clear:
+        if os.path.exists():
+            shutil.rmtree(Path(output_dir))
```

Comment:
```suggestion
        if output_dir.exists():
            shutil.rmtree(output_dir)
```

`Path` objects have an `exists()` method, and there is no need to wrap the shutil argument into a path (it already is!)